Wikipedia/UI-V5/WMFLocationManager.m

```diff
@@ -6,8 +6,6 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-static CLLocationDistance WMFMinimumDistanceBeforeUpdatingLocation = 1.0; //meters before we update location
-
 @interface WMFLocationManager ()<CLLocationManagerDelegate>
```

Comment:
The authorization stuff is mostly refactoring (i.e. moving logic out into its own methods instead of doing everything inline in `startMonitoring`)